src/renderers/shaders/ShaderChunk/occlusion_fragment.glsl.js

```diff
@@ -0,0 +1,26 @@
+export default /* glsl */`
+#ifdef USE_OCCLUSION
+    if (gl_FragColor.a > 0.0) {
+        int arrayIndex = 0;
+        vec2 depthUv;
+        if (gl_FragCoord.x>=depthWidth) {
```

Comment:
is this checking which eye to read the texture for?

Reply:
Yes. I considered passing it in as a value but that required deeper changes in the renderer.